_posts/2020-10-13-uni-retry.adoc

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: Mutiny - How does retry retry?
```

Comment:
You should create more drama: `How does retry... retries?`

Reply:
Good point! Fixed!